pkgs/applications/emulators/mame/default.nix

```diff
@@ -39,14 +38,14 @@ let
 in
 stdenv.mkDerivation rec {
```

Comment:
We could have switched to `finalAttrs` pattern here, maybe something to keep in mind at the next update.